### ISSUE TITLE
DataTable for registration data not showing value for select fields

### ISSUE THREAD

**Issue by MartinBenediktBusch** [lexicongovernance/pluraltools-frontend#426]
## Overview

Transfered `admin` issue: Select fields should show select option value not the id (see first screenshot).

After investigation of this issue I come to the conclusion that this is a `frontend` rather than an `admin` or `backend` issue. Here the reproduced bug as seem from the admin: 

![AdminDisplay](https://github.com/lexicongovernance/pluraltools-admin/assets/43137759/6f2236ff-d051-497c-af40-563a6c3209a9)

The issue is that for `SELECT` fields admin displays the `id` instead of the `value` of the select field. However, `admin` displays what it is supposed to display as can be seen from the database view: 

![Database](https://github.com/lexicongovernance/pluraltools-admin/assets/43137759/d7d17037-4634-488c-a90f-035fa35891b2)

However, also the `backend` does what it is supposed to do as it received the `id` instead of the `value` in case of a select field as can be seen from the network response of the frontend: 

![networkResponse](https://github.com/lexicongovernance/pluraltools-admin/assets/43137759/c5ff548c-33e9-4d0d-b2bf-4258b4248f53)

In summary, I think the propoer solution involves the `frontend` sending the value of the registration field option instead of the id in case the registration field is of type `SELECT`.

**Comment by diegoalzate** [comment]
pros of saving the actual value:
- simple to visualize after

cons of saving the actual value:
- hard to check what option the user selected from a value

the way the registration field options work is that they have a key and a value. the key is the id and the value is the actual string it represents. when saving the key it is easy to know what option the user stored.

if we look at this as just data and strings, it does make sense to store the actual value more than the id as a plain value.

**Comment by diegoalzate** [comment]
after discussing with @camilovegag we do want to go ahead with this transition but it requires changes on admin in order to make sense of previous selected values
